src/bootstrap/metrics.rs

```diff
@@ -124,6 +125,11 @@ impl BuildMetrics {
             }
         };
         invocations.push(JsonInvocation {
+            start_time: state
+                .invocation_start
+                .duration_since(std::time::SystemTime::UNIX_EPOCH)
+                .unwrap_or(Duration::ZERO)
```

Comment:
Will `Duration::ZERO` ever be useful? Why not panicking directly?

Reply:
Yeah, panicking seems fine. I don't think we should ever cross the epoch boundary in CI.